lib/livebook/settings.ex

```diff
@@ -117,4 +130,113 @@ defmodule Livebook.Settings do
   def set_update_check_enabled(enabled) do
     storage().insert(:settings, "global", update_check_enabled: enabled)
   end
+
+  @doc """
+  Gets a list of environment variables from storage.
+  """
+  @spec fetch_env_vars() :: list(EnvVar.t())
+  def fetch_env_vars do
+    for fields <- storage().all(:env_vars) do
+      struct(EnvVar, fields)
```

Comment:
```suggestion
      struct!(EnvVar, fields)
```

Reply:
Forgot to remove this coz I was having issues with already persisted env vars with old fields

Reply:
But I still need to remove the `id` field since it doesn't exists inside `EnvVar` embedded schema